retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/RetrofitMeterIdPrefixFunction.java

```diff
@@ -48,7 +48,7 @@
     /**
      * Returns a newly created {@link RetrofitMeterIdPrefixFunction} with the specified {@code name}.
      */
-    public static RetrofitMeterIdPrefixFunction of(String name) {
+    public static MeterIdPrefixFunction of(String name) {
```

Comment:
We can continue returning `RetrofitMeterIdPrefixFunction` if we make `RetrofitClassAwareMeterIdPrefixFunction` extend `RetrofitMeterIdPrefixFunction`. Let's make `RetrofitMeterIdPrefixFunction` non-final with package-private constructor and let `RetrofitClassAwareMeterIdPrefixFunction` extend it.

Reply:
This is not fixed yet.